server/src/main/scala/controllers/Application.scala

```diff
@@ -160,7 +160,15 @@ class Application @Inject()(implicit val config: Configuration, env: Environment
 
   val ctrl: DrtSystemInterface = DrtActorSystem.drtSystem
 
-  ctrl.run()
+  val log: LoggingAdapter = system.log
+
+  val systemStartGracePeriod: FiniteDuration = config.get[Int]("start-up-grace-period-seconds").seconds
+
+  log.info(s"Scheduling crunch system to start in ${systemStartGracePeriod.toString()}")
```

Comment:
Wondering if it would add any down time in case application is down before start.

Reply:
This is clever as it only delays the crunch system, not the end points. So the app is still up and responsive, but it'll wait before starting to ingest new data